homeassistant/components/switchbot_cloud/climate.py

```diff
@@ -70,11 +76,12 @@ class SwitchBotCloudAirConditioner(SwitchBotCloudEntity, ClimateEntity):
     ]
     _attr_fan_mode = FanState.FAN_AUTO
     _attr_hvac_modes = [
-        HVACMode.HEAT_COOL,
+        HVACMode.AUTO,
```

Comment:
I think this change is a breaking change, let's separate it

Reply:
I think it is fine as the state is not saved in the Switchbot API but I will rollback it